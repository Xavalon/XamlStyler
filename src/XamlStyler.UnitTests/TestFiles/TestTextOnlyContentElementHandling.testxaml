<Root>

	<!-- START: Test for TEXT-ONLY-CONTENT ELEMENT handling -->
  <EmptyNode></EmptyNode>
  <SingleSpace> </SingleSpace>
  <SingleTab> </SingleTab>
  <SingleAmp>&amp;</SingleAmp>
  <SingleLine>test here</SingleLine>
	<SingleLine a="a" b="b">test here</SingleLine>
	<SingleLineWithLinefeed a="a" b="b" c="c">

		line

	</SingleLineWithLinefeed>
  <MultipleSpaces>                                         </MultipleSpaces>
  <MultipleTabs>              </MultipleTabs>
  <MultiLine>first line  
  second line</MultiLine>
	<MultiLineMixed>
		first line  

  second line

		<Child />

		third line
	</MultiLineMixed>
	<!-- END: Test for TEXT-ONLY-CONTENT ELEMENT handling -->

</Root>