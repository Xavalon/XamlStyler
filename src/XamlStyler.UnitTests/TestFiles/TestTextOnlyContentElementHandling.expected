<Root>

  <!--  START: Test for TEXT-ONLY-CONTENT ELEMENT handling  -->
  <EmptyNode />
  <SingleSpace />
  <SingleTab />
  <SingleAmp>&amp;</SingleAmp>
  <SingleLine>test here</SingleLine>
  <SingleLine a="a" b="b">test here</SingleLine>
  <SingleLineWithLinefeed a="a"
                          b="b"
                          c="c">
    line
  </SingleLineWithLinefeed>
  <MultipleSpaces />
  <MultipleTabs />
  <MultiLine>
    first line
    second line
  </MultiLine>
  <MultiLineMixed>
    first line
    second line
    <Child />
    third line
  </MultiLineMixed>
  <!--  END: Test for TEXT-ONLY-CONTENT ELEMENT handling  -->

</Root>